pkg/bundle/resources.go

```diff
@@ -306,3 +324,52 @@ func readContent(ctx context.Context, progress *progress.Progress, base, name st
 
 	return files, nil
 }
+
+func parseValueFiles(base string, chart *fleet.HelmOptions) (parsedChart *fleet.HelmOptions, err error) {
+	parsedChart = chart
+	if len(chart.ValuesFiles) != 0 {
+		valuesMap, err := generateValues(base, chart)
+		if err != nil {
+			return nil, err
+		}
+		if parsedChart.Values == nil {
+			parsedChart.Values = &fleet.GenericMap{}
+		}
+		parsedChart.Values.Data = valuesMap
+	}
+
+	return parsedChart, nil
+}
+
+func generateValues(base string, chart *fleet.HelmOptions) (valuesMap map[string]interface{}, err error) {
+
+	valuesMap = make(map[string]interface{})
+	if chart.Values != nil {
+		valuesMap = chart.Values.Data
+	}
+	for _, value := range chart.ValuesFiles {
+		valuesByte, err := ioutil.ReadFile(base + "/" + value)
+		if err != nil {
+			return nil, err
+		}
+		tmpMap := make(map[string]interface{})
+		err = v2.Unmarshal(valuesByte, tmpMap)
```

Comment:
There are existing yaml package from k8s sigs.k8s.io/yaml. Can we just use this without introducing new yaml package?

Reply:
tweaked the parsing logic to use k8s yaml